include/alloca.h

```diff
@@ -0,0 +1,7 @@
+#ifndef ALLOCA_H
+#define ALLOCA_H
+
+void* alloca(u32);
+#define alloca  __builtin_alloca
```

Comment:
This is likely to have problems with GCC build in the future. Doesn't necessarily need to be addressed right now, but a comment may help.

Reply:
Looks like GCC supports it. Other compilers may not, but it should be fine.
https://github.com/zeldaret/mm/pull/310/files